inc/sitemaps/class-sitemap-image-parser.php

```diff
@@ -67,7 +67,14 @@ public function get_images( $post ) {
 			$images[] = $this->get_image_item( $post, $src, $title, $alt );
 		}
 
-		$unfiltered_images = $this->parse_html_images( $post->post_content );
+		/**
+		 * Filters the post content before it is parsed for images.
+		 *
+		 * @param string $content
+		 */
+		$content = apply_filters( 'wpseo_sitemap_content_before_parse_html', $post->post_content );
```

Comment:
I think `wpseo_sitemap_content_before_parse_html_image` would be a more descriptive name.